crates/ibc/src/mock/context.rs

```diff
@@ -668,6 +679,7 @@ impl RouterBuilder for MockRouterBuilder {
     }
 }
 
+// TODO: REMOVE BEFORE MERGING
```

Comment:
any action needed for this comment?

Reply:
oops, that will be removed in next PR now. Removed comment: [5c602f1](https://github.com/cosmos/ibc-rs/pull/417/commits/5c602f1fd504c7bd7a7453f913a09359f2457800)